sdk/monitor/monitor-opentelemetry/src/logs/handler.ts

```diff
@@ -55,14 +56,29 @@ export class LogHandler {
    * Start auto collection of telemetry
    */
   private _initializeInstrumentations() {
-    if (this._config.instrumentationOptions.bunyan?.enabled) {
+    const bunyanLogLevelEnv =
+      process.env.APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL || process.env.BUNYAN_LOG_LEVEL;
+    const winstonLogLevelEnv =
```

Comment:
Same for Winston we should not add specific instrumentation log level control via env variable, JS code could be used for further customization

Reply:
Makes sense, I'll remove the logging library specific configuration env vars and have all libraries controlled by the `APPLICATION_INSIGHTS_LOGGING_LEVEL` env var.